src/pairs/Player.py

```diff
@@ -1,10 +1,13 @@
 class Player:
-    def __init__(self, original_deck, points=0, hand_state=None):
+    def __init__(self, original_deck, points=0, hand_state=None, hand_capacity=None):
         self.original_deck = original_deck
         self.points = points
         if hand_state is None:
             hand_state = self.hand_state = {card_type: 0 for card_type in original_deck}
         self.hand_state = hand_state
+        if hand_capacity is None:
+            hand_capacity = self.hand_capacity = {card_type: 1 for card_type in original_deck}
```

Comment:
Fix to `hand_capacity = {card_type: 1 for card_type in original_deck}`

Reply:
done